### DISCUSSION TITLE
关于导出下载时显示“无权限”的求助

### DISCUSSION THREAD

**Discussion by Frogjiangs** [the1812/Bilibili-Evolved#1461] [提问 / Q&A] [answered]
bilibili是大会员
下载的idm,aria2，显示链接，复制，在idm点击“新建任务”，粘贴链接，点击确认
就会出现“服务器响应显示您没有权限下载此文件，备注HTTP/1.1.403 forbidden”
近期没有下载过1个视频，不会存在给我封了的情况，查找讨论好像也没有我的情况，求助！谢谢各位！

**Comment by the1812** [comment, marked as answer]
要点 IDM 按钮导出的, 用法常见问题里有写
![image](https://user-images.githubusercontent.com/26504152/104293728-4cfa6400-54f9-11eb-8a48-d473d1d8863f.png)

**Reply by Frogjiangs** [reply]
感谢您的答疑，已经可以下载了，可是现在有一个新问题，就是走的电影会有好几十个flv文件，请问应该如何合并呀

**Reply by the1812** [reply]
https://github.com/the1812/Bilibili-Evolved/issues/201
